plugins/quickicon/eos/src/Extension/Eos.php

```diff
@@ -0,0 +1,356 @@
+<?php
+
+/**
+ * @package                 Joomla.Plugin
+ * @subpackage              quickicon.eos
+ * @copyright               (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license                 GNU General Public License version 2 or later; see LICENSE.txt
+ * @phpcs                   :disable PSR1.Classes.ClassDeclaration.MissingNamespace
+ */
```

Comment:
```suggestion
/**
 * @package     Joomla.Plugin
 * @subpackage  Quickicon.eos
 *
 * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
 * @license     GNU General Public License version 2 or later; see LICENSE.txt
 */
```
format
file has namespace